Source/Plugins/Core/com.equella.core/scalasrc/com/tle/web/api/search/SearchHelper.scala

```diff
@@ -118,17 +121,20 @@ object SearchHelper {
 
   /**
     * Parse a string to a new instance of Date in the format of "yyyy-MM-dd".
-    * @param date The string to parse.
-    * @return An option which wraps an instace of Date.
+    * @param dateString The string to parse.
+    * @param time The time added to a date.
+    * @return An option which wraps an instance of Date.
```

Comment:
Be good to further detail the return value. That way a user can rely on their IDE's documentation feature to display useful guidance without needing to review the function.

```suggestion
    * @return An Option which wraps an instance of Date, combining the successfully parsed dateString and provided time (based on the system's default timezone).
```